docs/impact.md

```diff
@@ -48,19 +48,16 @@ A listing of the previous grants can be found [here](./collaboration/acknowledgm
 
 ### Datasets
 
-![](./assets/img/openneuro_data_growth.png)
+--8<-- "tmp/openneuro_data_growth.html"
+
```

Comment:
I haven't seen this before, google is telling me it's a mkdoc's snippet, but it's pointing to an htm file in a temp folder?